CHANGELOG.md

```diff
@@ -1,7 +1,7 @@
 
 ## 8.0.1
-#### 2022-08-19
-* Updates `nth-check` dependency to 2.0.1
+#### 2022-09-08
```

Comment:
Updating nth-check was not correctly fixing the vulnerability issue, we need to upgrade svgo because nth-check is introduced via svgo.